docs/src/dMCDA.md

```diff
@@ -1 +1,159 @@
+$$Q_i = \delta \frac{S_i-S^-}{S^+-S^-} + (1-\delta)\frac{R_i-R^-}{R^+-R^-}$$
+
+Here, $\delta$ is a weighting which balances strategy between maximising all criteria and regret due to the worst performing criteria. Additional steps to the algorithm give guidelines for choosing between a single best alternative and n (n>1) best alternatives.
+
+## Post-ranking distance sorting 
+If highly ranked sites happen to be very close together in a particular domain, this can represent an issue for site selection. This is because the site selection algorithm will seed or shade in a very small region, representing a significant loss of resources if a high damage, localised event, such as a ship grounding, were to occur. To represent an insurance policy against this possibility, the user can also specify distance thresholds for selected sites. If sites are selected which are too close, sites will be replaced by the next highest ranked sites which are further apart until the distance threshold is satisfied. The distance sorting parameter inputs are described in the table below:
+
+|Parameter                    |Description                                                                        |
+|-----------------------------|-----------------------------------------------------------------------------------|
+|```dist_thresh```            |Sites closer than (median distance between sites) - dist_thresh% will be replaced. |
+|```top_n```                  |Number of top ranked alternatives to select from when replacing sites.             |
+
+
+## Using site selection separately
+ADRIA's site selection algorithms can also be implemented outside of the ecological model, using the `run_site_selection()` function. The user provides data layers representing the selection criteria and weights to produce a set of site ranks for seeding and/or shading. An example of the function's usage is shown below. First an ADRIA domain is loaded, containing environmental data such as wave and heat stress layers and site data. Then a criteria data frame is created, which specifies criteria weightings and wave and dhw scenarios. Finally, coral cover layers for each scenario are loaded. These are used within `run_site_selection()` to generate reccommended intervention sites for seeding at time `ts`, for each parameter scenario.
+
+```julia
+using ADRIA
+using ADRIA: run_site_selection
+using DataFrames
+
+@info "Loading data package"
+here = @__DIR__
+dom = ADRIA.load_domain(joinpath(here, "Example_domain"))
+
+criteria_df = ADRIA.sample(dom, 5) # get scenario dataframe
+
+area_to_seed = 1.5 * 10^-6  # area of seeded corals in km^2
+ts = 5  # time step to perform site selection at
+
+# initial coral cover matching number of criteria samples (size = (no. criteria scens, no. of sites))
+sum_cover = fill(0.1, nrow(criteria_df), nrow(dom.site_data))
+ranks = run_site_selection(dom, criteria_df[criteria_df.guided.>0, :], sum_cover[criteria_df.guided.>0, :], area_to_seed, ts)
```

Comment:
Minor comment: we import ADRIA anyway, so you could save a line and call `ADRIA.run_site_selection` instead